lib/tasks/terraform.rake

```diff
@@ -0,0 +1,6 @@
+namespace :db do
+  desc 'tear down and rebuild the database'
+  task terraform: [:drop, :create, :migrate, :upsert_vita_partners] do
+    puts "database rebuilt!"
```

Comment:
Does this also need to insert states now?

Reply:
it does.